earthmover/operations/column.py

```diff
@@ -417,17 +443,20 @@ def execute(self, data: 'DataFrame', **kwargs) -> 'DataFrame':
             )
             raise
 
-        for _column in self.columns_list:
-            try:
-                data[_column] = (
-                    dask.dataframe.to_datetime(data[_column], format=self.from_format, exact=bool(self.exact_match), errors='coerce' if self.ignore_errors else 'raise')
-                        .dt.strftime(self.to_format)
-                )
-
-            except Exception as err:
-                self.error_handler.throw(
-                    f"error during `date_format` operation, `{_column}` column... check format strings? ({err})"
-                )
+        for col in self.columns_list:
+            # Use the fnmatch column functionality to find each valid value
+            for data_column in data.columns:
+                if fnmatch.fnmatch(data_column, col):
+                    try:
+                        data[data_column] = (
```

Comment:
We will need to run the datetime transform outside the wildcard loop in case multiple wildcards match the same column more than once.